app/concepts/base/repository.rb

```diff
@@ -20,7 +20,11 @@ def update_all(params)
           end
 
         def create(attrs)
-            adapter.create(attrs)
+            begin
+                adapter.create(attrs)
+            rescue ActiveRecord::RecordInvalid => error
+                ["Error": "NotFound"]
```

Comment:
`NotFound`?

Reply:
Gonna replace it. I deleted most of my If statements.